### PR TITLE
Completed Task1

### PR DESCRIPTION
**Pull request by shidenko97**

### CODE REVIEW

File: hidenko_serhii/01/Task1.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+user_pass, bot1_pass, bot2_pass = False, False, False  # Sign of missed
+
+user_points += randint(min_card, max_card) + randint(min_card, max_card)  # User's start points
+bot1_points += randint(min_card, max_card) + randint(min_card, max_card)  # Bot1's start points
+bot2_points += randint(min_card, max_card) + randint(min_card, max_card)  # Bot2's start points
+
+while not user_pass or not bot1_pass or not bot2_pass:  # While all players not missed
+
+    if user_points <= win_points:  # If user's points less or equal win
+        print("User current points: " + str(user_points))
```
Comment (medviediev) [inline hidenko_serhii/01/Task1.py]: If a body of an if expression is more then just one line of code, it should be picked out with blank lines

File: hidenko_serhii/01/Task1.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        bot2_pass = randint(0, 1)
+        if not bot2_pass:  # If bot2 not missed - adding card to points
+            bot2_points += randint(min_card, max_card)
+    else:  # Bot2's points more than win points
+        bot2_pass = True
+    print("Does bot 2 pass? " + str(bool(bot2_pass)))
+
+    print("----------------------")
+
+if win_points >= user_points > max_points:  # If user's points less or equal win points, but more than max
```
Comment (medviediev) [inline hidenko_serhii/01/Task1.py]: A bit wrong logic

File: hidenko_serhii/01/Task1.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
+from random import randint
+
+
+min_card = 2  # Min card value
```
[the remark this answers is not in the document]
Comment (shidenko97) [reply hidenko_serhii/01/Task1.py]: Yes, i forget

File: hidenko_serhii/01/Task1.py (hunk 1 above)
Comment (shidenko97) [reply hidenko_serhii/01/Task1.py]: Yes, i forget

File: hidenko_serhii/01/Task1.py (hunk 2 above)
Comment (shidenko97) [reply hidenko_serhii/01/Task1.py]: Maybe, i will think about it

**Pull request merged**